src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,10 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
You don't need to set margins and padding for all elements, reset only for tags that actually need this, for example div and span tags don't have any of them and so on

Reply:
I think as it is a small project I want to keep it as it wont impact on other things and I expect that I wont add anything more then. And by having margin 0 and padding 0 its easy for me to center elements correctly. So its just my point view.

Reply:
> I think as it is a small project I want to keep it as it wont impact on other things and I expect that I wont add anything more then. And by having margin 0 and padding 0 its easy for me to center elements correctly. So its just my point view.

You can reset margins and padding only for tags that actually have them, div and span tags don't have any of them and so on